src/inmanta/protocol/rest/server.py

```diff
@@ -140,6 +141,11 @@ async def _call(self, kwargs: Dict[str, str], http_method: str, call_config: com
                 else:
                     raise exceptions.UnauthorizedException("Access to this resource is unauthorized.")
 
+            except JSONDecodeError as e:
+                error_message = f"The request body couldn't be decoded as a JSON: {e}"
+                LOGGER.exception(error_message)
```

Comment:
I think this has to be logged on a lower log level than error, as it is not a server error (could be info or even debug)